test/test_client.py

```diff
@@ -1837,22 +1837,22 @@ def test_handshake_08_invalid_aws_ec2(self):
         )
 
     def test_dict_hints(self):
-        c = rs_or_single_client()
+        c = self.client
         try:
             c.t.t.find(hint={"x": 1})
         except Exception:
             self.fail("passing a dictionary hint to find failed!")
 
     def test_dict_hints_sort(self):
-        c = rs_or_single_client()
+        c = self.client
         try:
             result = c.t.t.find()
```

Comment:
Can we always ensure it's possible to pass a dict sort to the find() call itself: `find(sort=...)`?